### DISCUSSION TITLE
Script to clone a sliced file, with multiple exposure settings

### DISCUSSION THREAD

**Discussion by ccatlett1984** [sn4k3/UVtools#372] [Scripts]
https://github.com/ccatlett1984/Dumping_Ground/blob/main/modify_exposures.ps1

Uses a windows form for the file selection dialog, as well as if it is unable to locate the UVTools.Core.dll file.
Prompts for input of:
Max Exposure
Step Value (Decrease of Exposure per step)
Number of Iterations

It saves the output files in the same directory as the original sliced file, and appends the exposure value to the end of the filename.

I have found this useful for generating a range of calibration prints, for dialing in my exposures with different resins.

**Comment by sn4k3** [comment]
Good job 👍
Just a few notes:

1. Look at https://github.com/sn4k3/UVtools/blob/master/UVtools.Core/FileFormats/FileFormat.cs#L881
2. Those should be outside the Loop since they are the same value every iteration:
```C#
$filePath = [System.IO.Path]::GetDirectoryName($inputFile)
$fileExt = [System.IO.Path]::GetExtension($inputFile)
$fileNoExt = [System.IO.Path]::GetFileNameWithoutExtension($inputFile)
```
